handlers/deposit_resource.go

```diff
@@ -57,6 +57,7 @@ func (d *depositResource) Handle(params operations.DepositResourceParams) middle
 func (d *depositResource) loadParams(resourceID string, data models.Resource) datautils.Resource {
 	resource := datautils.NewResource(data.(map[string]interface{}))
 	resource["id"] = resourceID
+	resource["version"] = 1
```

Comment:
This should also set currentVersion = true (per https://github.com/sul-dlss-labs/taco/wiki/TACO-Internal-Processing-Steps#deposit-resource).

Reply:
@aaron-collier that's ticket #290. This PR is for issue #337

Reply:
👍